source/gwb-grid/main.cc

```diff
@@ -247,14 +248,28 @@ int main(int argc, char **argv)
 
   try
     {
+      if (find_command_line_option(argv, argv+argc, "-v") || find_command_line_option(argv, argv+argc, "--version"))
+        {
+          std::cout << "World Builder Grid Visualization tool.\n"
+                    << "GWB Version: " << WorldBuilder::Version::MAJOR << "."
+                    << WorldBuilder::Version::MINOR << "."
+                    << WorldBuilder::Version::PATCH << "."
+                    << WorldBuilder::Version::LABEL
+                    << std::endl;
```

Comment:
while we are at it, can you also add the git hash?

Reply:
done

Reply:
Thanks!